var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -58,12 +58,21 @@ class Gromacs(CMakePackage):
                     'AVX2_128', 'AVX2_256', 'AVX_512', 'AVX_512_KNL',
                     'IBM_QPX', 'Sparc64_HPC_ACE', 'IBM_VMX', 'IBM_VSX',
                     'ARM_NEON', 'ARM_NEON_ASIMD'))
-    variant('rdtscp', default=True, description='Enable RDTSCP instruction usage')
+    variant('rdtscp', default=True,
+            description='Enable RDTSCP instruction usage')
```

Comment:
This seems awfully specific.  Is this an option just because it's supposed to be disabled on non-x86 machines?  If so, I think it makes sense to remove the variant and add that logic below instead of requiring the user to know about this option, e.g.:

```python
options.append('-DGMX_USE_RDTSCP:BOOL=%s' % ('target=x86_64' in self.spec))
```

Reply:
I think it works on `i686` as well.

Reply:
I introduced this option because I admin a cluster which has on one partition Opteron 6174 processors and Xeon L5420 on another. The latter does not support rdtscp but both support SSE2.